### ISSUE TITLE
20230331 Simulation Documentation URL and SRW Notes

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#5379]
The Simulation Documentation URL will change to a link, not an editable field. If a simulation has a documentationURL, the link should be rendered as "Simulation Documentation" (drop the URL). This should open a new tab with the documentationURL.

Only examples can have a documentationURL. The field will not be editable.  There are a few existing entries in non-examples, but these can be ignored.

On SRW there's a "Notes" link which goes to the Simulation Documentation URL. This confuses the Notes feature, which allows users to create notes about simulations. Therefore, this will be removed in the React UI.

**Comment by garsuga** [comment]
Just to confirm, should I change documentationURL on Angular as well or only on React?

**Comment by robnagler** [comment]
Per the meeting. We'll wait on React until we have to do SRW. In Angular, change it to just be a link, but this is lower priority than other issues.
